config/initializers/secure_headers.rb

```diff
@@ -16,9 +16,10 @@
       :report_only => false,
       :default_src => ["'self'"],
       :frame_src   => ["'self'"],
-      :font_src    => ["'self'", 'https://fonts.gstatic.com'],
-      :connect_src => ["'self'"],
-      :style_src   => ["'unsafe-inline'", "'self'"],
+      :font_src    => ["'self'", 'https://fonts.gstatic.com',"https://fonts.googleapis.com"],
+      :img_src   => ["'self'", "data:"],
+      :connect_src => ["'self'","'http'","'https:'","'ws:'"],
```

Comment:
Looks like there were issues with safari and `'self'` not working for websockets. https://github.com/w3c/webappsec-csp/issues/7

I do worry that `'http'` points to every website out there. (probably covers `http` and `https`) but more importantly, it is a substring of every website, so we loose this protection